tutorials/ensembles-stacking/ensembles-stacking.R

```diff
@@ -111,166 +111,119 @@ h2o.removeAll() # Clean slate - just in case the cluster was already running
 #First, import a sample binary outcome train and test set into the H2O cluster.
 train <- h2o.importFile(path = normalizePath("../data/higgs_10k.csv"))
```

Comment:
use real paths for train and test datasets
for example:
train <- h2o.importFile("https://s3.amazonaws.com/h2o-public-test-data/smalldata/testng/higgs_train_5k.csv")
test <-  h2o.importFile("https://s3.amazonaws.com/h2o-public-test-data/smalldata/testng/higgs_test_5k.csv")